src/Session.php

```diff
@@ -550,4 +555,72 @@ protected function save()
 
         return $this->getStore()->save($this->getId(), $values, $this->ttl);
     }
+
+    /**
+     * Put a key / value pair or array of key / value pairs in the session.
+     *
+     * @param  string|array  $key
+     * @param  mixed       $value
+     * @return void
+     */
+    public function put($key, $value = null)
+    {
+        if (! is_array($key)) {
+            $key = [$key => $value];
```

Comment:
Short array syntax is great, but it's available from PHP 5.4 and I'm afraid we need to support older versions.